dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/registry/ServerNodeManager.java

```diff
@@ -153,29 +153,15 @@ class WorkerNodeInfoAndGroupDbSyncTask implements Runnable {
         @Override
         public void run() {
             try {
-
                 // sync worker node info
                 updateWorkerNodes();
                 updateWorkerGroupMappings();
-                notifyWorkerInfoChangeListeners();
```

Comment:
Why you remove this?  this is used to notifier the `LowerWeightHostManager`

Reply:
> Why you remove this? this is used to notifier the `LowerWeightHostManager`

Because `updateWorkerGroupMapping` has notify(has invoke `notifyWorkerInfoChangeListener`). So we do not need to repeat it a second time.

Reply:
It's better to remove the method in `updateWorkerGroupMappings`

Reply:
> It's better to remove the method in `updateWorkerGroupMappings`

but others also will invoke `updateWorkerGroupMappings`, such as spring bean init. May `notifyWorkerInfoChangeListener` keep in `updateWorkerGroupMappings` should be more clear. WDYT.
![image](https://user-images.githubusercontent.com/20518339/203802201-418c5cfb-1d29-4eab-93c1-4499ec801513.png)

Reply:
so i move all update worker node info methods to a private methods.